### ISSUE TITLE
Compile error with latest commit.

### ISSUE THREAD

**Issue by fade2gray** [cheat-engine/cheat-engine#2397]
[add dark mode to the virtualstringtree component]( https://github.com/cheat-engine/cheat-engine/commit/96ee22c9019c0d5a9e304c45ab75e42f4e2545c7)

```
Compile Project, Mode: Release 64-Bit, Target: bin\cheatengine-x86_64.exe: Exit code 1, Errors: 1
newvirtualstringtree.pas(8,32) Fatal: Cannot find laz.VirtualTrees used by newvirtualstringtree. Check if package laz.virtualtreeview_package is in the dependencies.

Compile Project, Mode: release 64-Bit, Target: C:\Users\fade2gray\Desktop\CE Sources\CE\cheat-engine-master\Cheat Engine\bin\tutorial-x86_64.exe: Exit code 1, Errors: 1
newvirtualstringtree.pas(8,32) Fatal: Cannot find laz.VirtualTrees used by newvirtualstringtree. Check if package laz.virtualtreeview_package is in the dependencies.
```

**Comment by avboy1337** [comment]
the same error

**Comment by a597407455** [comment]
the same error

**Comment by vladbakanov** [comment]
Error

**Comment by zhang2651** [comment]
the same error

**Comment by manyougz** [comment]
Seems to be a bug with version 7.4.2

**Comment by cheat-engine** [comment]
i'll fix it when i'm done with the mapping the most important virtualstringtree properties and methods to lua
Hint: Read the error message. It tells you what to do

**Comment by ENDDOGEINGTON** [comment]
I don't get it help pls

**Comment by ENDDOGEINGTON** [comment]
i have same error please help

**Comment by DGHades** [comment]
Go to "Project" in the top left and then click "Project Inspector".
Click on the Add button with the plus.
Select the page New Requirement.
Write "laz.virtualtreeview_package" into the "Package-Name" field.
Click Ok.

**Comment by fade2gray** [comment]
@cheat-engine 
> i'll fix it when i'm done with the mapping the most important virtualstringtree properties and methods to lua
Hint: Read the error message. It tells you what to do

One year later and still waiting 😉

**Comment by Kbmobi** [comment]
I don't think so. Tried both master and 7.5. I see virtualtreeview_package, but not laz.virtualtreeview_package.

**Comment by cheat-engine** [comment]
https://github.com/cheat-engine/cheat-engine/blob/master/Cheat%20Engine/cheatengine.lpi#L817

**Comment by jlanawalt** [comment]
I tried the latest, seemed to have thee same issue, despite laz.virtualtreeview_package seeming to already be in the requirements. I worked around the issue by commenting out the if/else block and the offending TVirtualStringTree line in the else block so it used TCustomVirtualStringTree instead. I don't know if I need that code or not. It seems to do the trick for getting it to compile and run and work for what I was after.

**Comment by fade2gray** [comment]
No errors this way, compiling the latest source using either Lazarus 2.2.2 or 2.26.


https://github.com/cheat-engine/cheat-engine/assets/1332009/835b2a61-5c94-44d8-ba6e-019e8e302601

**Comment by whiteCupz** [comment]
how to download the old version lazarus

**Comment by fade2gray** [comment]
https://sourceforge.net/projects/lazarus/files/Lazarus%20Windows%2064%20bits/

**Comment by shy113** [comment]
https://wiki.freepascal.org/VirtualTreeview_
![image](https://github.com/user-attachments/assets/3d6cb8ca-4bb5-4b8f-bef7-41e4f2ede9a4)

Hello everyone, I also encountered this problem. I found the reason for this problem on this website,[](https://wiki.freepascal.org/VirtualTreeview_
)

**Comment by unk010** [comment]
> Go to "Project" in the top left and then click "Project Inspector". Click on the Add button with the plus. Select the page New Requirement. Write "laz.virtualtreeview_package" into the "Package-Name" field. Click Ok.

thank you

**Comment by slooi** [comment]
> Go to "Project" in the top left and then click "Project Inspector". Click on the Add button with the plus. Select the page New Requirement. Write "laz.virtualtreeview_package" into the "Package-Name" field. Click Ok.

Anyone else get the following error after adding the virtualtreeview pacakge when compiling the tutorial? `bettercontrols.pas(109,68) Fatal: Cannot find mainunit2 used by betterControls of the Project Inspector.`

**Comment by fade2gray** [comment]
@slooi You don't say which version of Lazarus you are using, but whichever it is, it's probably the wrong one.

https://github.com/cheat-engine/cheat-engine#basic-build-instructions

![Screenshot 2024-01-18 164529](https://github.com/cheat-engine/cheat-engine/assets/1332009/f15d44a8-1c6c-49c6-ad29-f8c614b35afe)

**Use Lazarus 2.2.2 or Lazarus 2.2.6.**
